fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/socialmedia/SocialMediaViewModel.java

```diff
@@ -204,34 +201,26 @@ public void subscribeToGeneralChannel(String laoId) {
   public void subscribeToChannel(String laoId) {
     Log.d(TAG, "subscribing to channel: " + ROOT + laoId + "/social/<sender>");
 
-    try {
-      String channel = ROOT + laoId + "/social/" + mKeyManager.getMainPublicKey().getEncoded();
-
-      Disposable disposable =
-          mLaoRepository
-              .sendSubscribe(channel)
-              .subscribeOn(Schedulers.io())
-              .observeOn(AndroidSchedulers.mainThread())
-              .timeout(5, TimeUnit.SECONDS)
-              .subscribe(
-                  answer -> {
-                    if (answer instanceof Result) {
-                      Log.d(TAG, "subscribed to the channel");
-                    } else {
-                      Log.d(TAG, "failed to subscribe to the channel");
-                    }
-                  },
-                  throwable ->
-                      Log.d(TAG, "timed out waiting for result on subscribe/channel", throwable));
-
-      disposables.add(disposable);
-
-    } catch (GeneralSecurityException | IOException e) {
-      Log.d(TAG, PK_FAILURE_MESSAGE, e);
-      Toast.makeText(
-              getApplication().getApplicationContext(), PK_FAILURE_MESSAGE, Toast.LENGTH_SHORT)
-          .show();
-    }
+    String channel = ROOT + laoId + "/social/" + mKeyManager.getMainPublicKey().getEncoded();
```

Comment:
the social media uses the PoP token for the channel

Reply:
I want yo uto merge your PR before applying this change as it might create conflicts. But as you want